ws/api/transport.go

```diff
@@ -47,7 +52,7 @@ func MakeHandler(svc ws.Service, tc mainflux.ThingsServiceClient, l log.Logger)
 	logger = l
 
 	mux := bone.New()
-	mux.GetFunc("/channels/:id/messages", handshake(svc))
+	mux.GetFunc("/channels/*", handshake(svc))
```

Comment:
Can we use `/channels/:id/*` here?

Reply:
Maybe even something like `channels/:id/messages(/)?.*`

Reply:
Yes :id parts can be used, but with the current implementation id and subtopic are managed on the same way by regexp, otherwise you get id from GetValue and subtopic form regexp, or we use regexp for both but ignore :id definition on router. What did you think?

Reply:
@beres I think that it would be more readable if we get id from `GetValue` and subtopic using regex.

Reply:
what did you think about something like:
	mux.GetFunc("/channels/:id/messages", handshake(svc))
	mux.GetFunc("/channels/:id/messages/{:subtopic[a-zA-Z0-9/-_*>]+}", handshake(svc))

Reply:
I dont think this will work, have you tried this?

Reply:
@mteodor you are right, it does not actually work with bone mux :see_no_evil:

Reply:
@beres did you actually test this implementation thoroughly, or you plan to break 0.8 for huge number of companies using Mainflux :)

Reply:
@drasko no, i believed  something like that could be possible, but as i replyed to @mteodor with bone does not work, while on coap mux it works. 

This seems to be the more readable solutions base on your preferences
mux.GetFunc("/channels/:id/messages", handshake(svc))
mux.GetFunc("/channels/:id/messages/*", handshake(svc))

Reply:
Committing to Mainflux at this mature stage is big responsability, especially when change is so big and touches core.

I suggest that you bring back WIP label to this PR until you test everything manually very throughly.

We are planning the 0.8 release with this change for next Friday, we will not have so much time to find potential regression.

Reply:
Just foe the record, I really appreciate your work here, and you are helping us a lot with taking this issue.

Reply:
@drasko I probably did not explain it well before, it was a proposal. If you liked it, I implemented it, otherwise not.
I have tested all the changes anyway it's right you take the time you need. :)

Reply:
@beres we like it. Just need to put it on the quality level of Mainflux.